resources/js/Pages/Lessons/Show.vue

```diff
@@ -0,0 +1,120 @@
+        let gameMessage = response.data['gameLoaded'];
+
+        if (response.data['gameLoaded'] == null) {
+            gameInstance.SendMessage('GameController', 'newGame');
+        } else {
+            gameInstance.SendMessage('GameController', 'loadedDataFromWeb', gameMessage);
+        }
+    } catch (e) {
+        gameInstance.SendMessage('GameController', 'newGame');
+        console.log("axios error: " + e);
+    }
+}
+
+</script>
+<template>
+    <Head title="GameView"/>
+    <div class="">
+        <div id="unity-container" class="unity-desktop flex flex-col">
+            <canvas ref='canvas' id="unity-canvas" width=960 height=600
+                    class="border border-solid border-8 border-blue-400 rounded-3xl">
+            </canvas>
+            <div id="unity-loading-bar">
+                <div id="unity-logo"></div>
+                <div id="unity-progress-bar-empty">
+                    <div id="unity-progress-bar-full"></div>
+                </div>
+            </div>
+            <Transition
+                enter-active-class="transition-opacity duration-300"
+                enter-from-class="opacity-0"
+                enter-to-class="opacity-100"
+                leave-active-class="transition-opacity duration-150"
+                leave-from-class="opacity-100"
+                leave-to-class="opacity-0"
+            >
+                <div v-if="showChallengeWindow">
+                    <form @submit.prevent="checkAnswer" class="mb-6 flex flex-col bg-zinc-500 p-3 ">
+                        <label for="large-input" class=" mb-2 text-lg font-medium text-gray-900 dark:text-gray-300">
+                            Answer here
+                        </label>
+                        <div v-show="resultWarning">
+                            <div
+                                class="flex p-4 mb-4 text-sm text-red-700 bg-red-100 rounded-lg dark:bg-red-200 dark:text-red-800"
+                                role="alert">
+                                <svg aria-hidden="true" class="flex-shrink-0 inline w-5 h-5 mr-3" fill="currentColor"
+                                     viewBox="0 0 20 20"></svg>
+                                <div>
+                                    <span class="font-medium">Attempt Incorrect! </span>
+                                    Change a few things up and try submitting again.
+                                </div>
+                            </div>
+                        </div>
+                        <textarea v-model="challengeUserAnswer" type="text" id="large-input"
+                                  class="block  h-40 p-4 w-full text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/>
```

Comment:
```suggestion
                 <textarea
                    v-model="challengeUserAnswer"
                    type="text"
                    id="large-input"
                    class="block  h-40 p-4 w-full text-gray-900 bg-gray-50 rounded-lg border border-gray-300 sm:text-md focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
                />
```
This can be formatted as the transition tag :)